inference-engine/ie_bridges/python/src/openvino/offline_transformations/offline_transformations_api_impl.hpp

```diff
@@ -4,13 +4,17 @@
 
 #pragma once
 
+#include <string>
+
 #include "Python.h"
 #include "ie_api_impl.hpp"
 
 namespace InferenceEnginePython {
 
 void ApplyMOCTransformations(InferenceEnginePython::IENetwork network, bool cf);
 
+void ApplyPOTTransformations(InferenceEnginePython::IENetwork network, std::string device);
```

Comment:
1. Why POT transformations are parametrized with device? Should not they be device agnostic?
2. Why POT needs these transformations? is it supposed that part of the POT is written in ngraph and this is a way to do this?

Reply:
Hi,
1. For specific devices POT need special offline transformations to quantize the whole model (e.g., LSTMCells). These transformations will be applied depending on the target device and frees the user from including these transformations, for example, at the graph conversion stage via MO. If executed on other devices, the converted subgraphs will be executed in the reference mode.
2. POT will apply this transformation before loading IR to the toolkit. After transformation will be applied, POT will serialize the nGraph to IR and start to compress it using MO representation.